test/aspect/aspect.bzl

```diff
@@ -52,7 +59,11 @@ def _rule_impl(ctx):
 
 aspect_test = rule(
     implementation = _rule_impl,
-    attrs = { "scala_rule" : attr.label(aspects = [test_aspect]) },
+    attrs = {
+        "scala_test" : attr.label(aspects = [test_aspect]),
+        "scala_library" : attr.label(aspects = [test_aspect]),
```

Comment:
It took me some time to understand the motivation for these attributes.
WDYT about renaming them to include some context? maybe `scala_library_test_target` etc